plugins/PostProcessingPlugin/scripts/ColorMix.py

```diff
@@ -0,0 +1,190 @@
+# Written by John Hryb - john.hryb.4@gmail.com
+
+##history / change-log:
+##V1.0.0
+
+## Uses -
+## M163 - Set Mix Factor 
+## M164 - Save Mix - saves to T3 as a unique mix
+
+import re #To perform the search and replace. 
+from ..Script import Script
+
+class ColorMix(Script):
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name":"ColorMix 2-1",
+            "key":"ColorMix 2-1",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "a_unitsOfMeasurement":
+                {
+                    "label": "Units of measurement",
+                    "description": "Input value as mm or layer number.",
+                    "type": "enum",
+                    "options": {"mm":"mm","layer":"Layer"},
+                    "default_value": "layer"
+                },
+                "b_start_height":
+                {
+                    "label": "Start Height",
+                    "description": "Value to start at (mm or layer)",
+                    "type": "float",
+                    "default_value": 0,
+                    "minimum_value": "0"
+                },
+                "c_behavior":
+                {
+                    "label": "Fixed or blend",
+                    "description": "Select Fixed (set new mixture) or Blend mode (dynamic mix)",
+                    "type": "enum",
+                    "options": {"fixed_value":"Fixed","blend_value":"Blend"},
+                    "default_value": "fixed_value"
+                },
+                "d_finish_height":
+                {
+                    "label": "Finish Height",
+                    "description": "Value to stop at (mm or layer)",
+                    "type": "float",
+                    "default_value": 0,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0.1",
+                    "enabled": "c_behavior == 'blend_value'" 
+                },
+                "e_mix_start":
+                {
+                    "label": "Start mix ratio",
+                    "description": "First extruder percentage 0-100",
+                    "type": "float",
+                    "default_value": 100,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0",
+                    "maximum_value_warning": "100"
+                },
+                "f_mix_finish":
+                {
+                    "label": "End mix ratio",
+                    "description": "First extruder percentage 0-100 to finish blend",
+                    "type": "float",
+                    "default_value": 0,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0",
+                    "maximum_value_warning": "100",
+                    "enabled": "c_behavior == 'blend_value'"
+                },
+                "g_notes":
```

Comment:
This setting is never used. Why do you need it?

In case you need it, I suggest to change the name:
```suggestion
                "notes":
```